packages/database/schema.prisma

```diff
@@ -673,6 +673,7 @@ model Insight {
   description      String
   vector           Unsupported("vector(512)")?
   documentInsights DocumentInsight[]
+  archived         Boolean                     @default(false)
```

Comment:
I think the 'archive' approach needs more refinement. Currently insights don't reappear. It's not ideal if someone complained about app performance a year ago, I archived it, and now people are complaining again (even in a different survey) and I don't see it in the environment insights.